lib/send_text.rb

```diff
@@ -0,0 +1,41 @@
+require 'twilio-ruby'
+require 'envyable'
+Envyable.load('./config/env.yml', 'development')
+
+class SendText
+
+  attr_reader :customers
+
+  def confirmation_text
+    sender
+  end
+
+  private
+
+    def sender
+
+      accountSID = ENV[accountSID]
+      authToken = ENV[authToken]
```

Comment:
Style/VariableName: Use snake_case for variable names. (https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)